sed/config/flash_example_config.yaml

```diff
@@ -3,20 +3,26 @@
 core:
   # defines the loader
   loader: flash
-
   # the beamline where experiment took place
   beamline: pg2
-
-  # The paths to the raw and parquet data directories.
+  # the instrument used
+  instrument: hextof
```

Comment:
We dont really use the instrument parameter anymore

Reply:
removed